def/de/menu_linux.cson

```diff
@@ -1,133 +1,133 @@
 Menu:
   "&File":
-    value: "File(&F)"
+    value: "Datei"
```

Comment:
You are right, I have tried to find characters inside the german words, matching the correct hotkey... but If I didnt find one, I have missed to integrate the correct hotkey in brackets. Will rework this.